redux/actions/utils.js

```diff
@@ -1,12 +1,18 @@
-export const getUrl = () => {
-  let _BASE_URL;
-  if (process.env.NODE_ENV === 'production') {
-    _BASE_URL = window.location.origin === 'https://app.niyonapp.com'
-      ? 'https://niyon-prod.herokuapp.com/api'
-      : 'https://niyon-staging.herokuapp.com/api';
-  } else {
-    _BASE_URL = 'https://niyon-dev.herokuapp.com/api';
-  }
+// const getUrl = () => {
+//   let _BASE_URL;
+//   if (process.env.NODE_ENV === 'production') {
+//     _BASE_URL =
+//       window.location.origin === 'https://app.niyonapp.com'
+//         ? 'https://niyon-prod.herokuapp.com/api'
+//         : 'https://niyon-staging.herokuapp.com/api';
+//   } else {
+//     _BASE_URL = 'https://niyon-dev.herokuapp.com/api';
+//   }
+
+//   return _BASE_URL;
+// };
 
+export const getUrl = () => {
+  const _BASE_URL = 'https://niyon-dev.herokuapp.com/api';
```

Comment:
the same here

Reply:
At the moment we can't because the function that Matt created for the environment doesn't work on the social login functionality. This is causing our deployments to fail, we've temporarily hardcoded it until the issue is fixed.